src/auto-trace.js

```diff
@@ -27,7 +27,7 @@ export function removeAllGlobalMiddlewares(){
  * @returns {Error}
  */
 export function asyncStacktrace(callback = ()=>{}, extraContext) {
-	const asyncStacktraceErr = new Error();
+	const asyncStacktraceErr = createError();
```

Comment:
So for these errors, we want to get rid of the frame for createError plus get rid of the frame for asyncStacktrace? So a total of two frames to get rid of, which is one "extra" frame? And since 1 is the default value for extra frames, we just don't provide an argument to createError?

👍

Reply:
Yep, that's correct. I figured that since createError() is a private function that one extra frame was a good default value (since it will always be called by another function in auto-trace). Helper functions will need to pass in a custom value (in most cases '2')